configure.ac

```diff
@@ -556,6 +556,11 @@ if test x"$GCC" = xyes; then
 		# The runtime code does not respect ANSI C strict aliasing rules
 		CFLAGS="$CFLAGS -fno-strict-aliasing"
 
+		# Compliant C++ unwinding code will stop at any frame with CANTUNWIND in the
+		# FDE. If we don't include -fexceptions, some of the runtime functions end up 
+		# having the CANTUNWIND attribute. This is necessary for the C++ EH backend.
+		CFLAGS="$CFLAGS -fexceptions"
```

Comment:
It looks like what happens is that we're actually removing an attribute, "nounwind". I was informed on #gcc on freenode that enabling exceptions on C code is supposed to be the default, otherwise interop with C++ unwinding breaks. I didn't notice significant size changes, but I can do a comparison if you'd like.